packages/eslint-plugin-pf-codemods/lib/rules/v5/popover-remove-props.js

```diff
@@ -0,0 +1,169 @@
+const { getPackageImports } = require('../../helpers');
+
+// https://github.com/patternfly/patternfly-react/pull/8201
+module.exports = {
+  meta: {
+    fixable: "code"
+  },
+  create: function(context) {
+    const imports = getPackageImports(context, '@patternfly/react-core')
+        .filter(specifier => ['Popover'].includes(specifier.imported.name));
+    const popoverImport = imports.find(imp => imp.imported.name === 'Popover');
+
+    return !popoverImport ? {} : {
+      JSXElement(node) {
+        if (popoverImport.local.name === node.openingElement.name.name) {
+          const boundaryAttr = node.openingElement.attributes.find(attribute => attribute.name.name === 'boundary');
+          const tippyAttr = node.openingElement.attributes.find(attribute => attribute.name.name === 'tippyProps');
+          const shouldCloseAttr = node.openingElement.attributes.find(attribute => attribute.name.name === 'shouldClose');
+          const onHiddenAttr = node.openingElement.attributes.find(attribute => attribute.name.name === 'onHidden');
+          const onHideAttr = node.openingElement.attributes.find(attribute => attribute.name.name === 'onHide');
+          const onMountAttr = node.openingElement.attributes.find(attribute => attribute.name.name === 'onMount');
+          const onShowAttr = node.openingElement.attributes.find(attribute => attribute.name.name === 'onShow');
+          const onShownAttr = node.openingElement.attributes.find(attribute => attribute.name.name === 'onShown');
+
+
+          if (boundaryAttr) {
+            context.report({
+              node,
+              message: "Popover boundary prop has been removed.",
+              fix(fixer) {
+                const fixes = [];
+                fixes.push(fixer.replaceText(boundaryAttr, ''));
+                return fixes;
+              }
+            });
+          }
+
+          if (tippyAttr) {
+            context.report({
+              node,
+              message: "Popover tippyProps prop has been removed.",
+              fix(fixer) {
+                const fixes = [];
+                fixes.push(fixer.replaceText(tippyAttr, ''));
+                return fixes;
+              }
+            });
+          }
+
+          if(shouldCloseAttr) {
+            const params = shouldCloseAttr.value.expression.params;
+            if(params.length === 2) { // only attempt to remove first param if 2 are passed
+              const firstParam = params[0];
+              context.report({
+                node,
+                message: "Popover shouldClose function's first parameter has been removed.",
+                fix(fixer) {
+                  const fixes = [];
+                  fixes.push(fixer.removeRange([firstParam.range[0], firstParam.range[1] + 1])); // remove first property and following comma
```

Comment:
```suggestion
                  fixes.push(fixer.removeRange([firstParam.range[0], params[1].range[0]])); // remove first property up to start of second
```

I think this way would be more flexible (in case users have weird spacing around a comma for example) and also not leave the white space